### ISSUE TITLE
Human corpses don't retain their equipment

### ISSUE THREAD

**Issue by cu-kai** [Unvanquished/Unvanquished#3220]
When you die, although you retain your equipment while the death animation is playing, once that is finished, you have no equipment attached, for example the jetpack or the radar.

**Comment by slipher** [comment]
Duplicate of https://github.com/Unvanquished/Unvanquished/issues/206.
